generators/gulp/templates/gulpfile.js

```diff
@@ -1,12 +1,13 @@
+<% if (!babel) { -%>
```

Comment:
We add `'use strict'` everywhere. Babel or not because es6 didn't remove the strict mode.

It's only babel who add it by default, and that's a mistake on their part.